keystone/common/policies/project.py

```diff
@@ -65,6 +65,8 @@
     '(role:admin and domain_id:%(target.project.domain_id)s)'
 )
 
+RULE_CLOUD_ADMIN = 'rule:cloud_admin'
```

Comment:
this should be changed to allow service users to do it too by default. Something like `'rule:cloud_admin or rule:service_role'`